{"name": "用户输入运算数据a、b, 程序输出正确的结果", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'data/add_data.yaml'", "trace": "request = <SubRequest 'cmd_option' for <Function test_div[10-5-2]>>\n\n    @pytest.fixture(scope=\"session\", autouse=True)\n    def cmd_option(request):\n        \"\"\"\n        查看指定测试用例的测试数据\n        :param request:  参数化\n        :return:\n        \"\"\"\n        global data_path\n        my_file_path = request.config.getoption(\"--show\", default=\"add_data\")\n        if my_file_path == \"add_data\":\n            data_path = r\"data/add_data.yaml\"\n        elif my_file_path == \"range_in_add_data\":\n            data_path = r\"data/add_data_range_in.yaml\"\n        elif my_file_path == \"range_out_data\":\n            data_path = r\"data/data_range_out.yaml\"\n        elif my_file_path == \"type_error_data\":\n            data_path = r\"data/data_type_error.yaml\"\n        elif my_file_path == \"div_data\":\n            data_path = r\"data/div_data.yaml\"\n        elif my_file_path == \"rang_in_div_data\":\n            data_path = r\"data/div_data_rang_in.yaml\"\n        elif my_file_path == \"zero_division_error\":\n            data_path = r\"data/zero_division_error.yaml\"\n    \n>       with open(data_path, \"r\", encoding=\"utf-8\") as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'data/add_data.yaml'\n\n..\\conftest.py:83: FileNotFoundError"}, "description": "\n        测试add()函数\n        :param a: 用户输入的数据a\n        :param b: 用户输入的数据b\n        :param expected: 输出a加b的结果\n        :return: 返回a加b的结果\n        ", "parameters": [{"name": "a", "value": "10.0"}, {"name": "b", "value": "5.123456789012345"}, {"name": "expected", "value": "15.123456789012344"}], "start": 1715221620970, "stop": 1715221620970, "uuid": "c4cd9761-0234-47b5-a88c-d6998b46a1c2", "historyId": "68667095f4bfe66caf689e6c1851056f", "testCaseId": "b716330b98d44fcafb992142fcf6b5d2", "fullName": "test_case.test_case_01.TestCase#test_add", "labels": [{"name": "tag", "value": "run(order=2)"}, {"name": "tag", "value": "test_normal"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_case_01"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "wangmin"}, {"name": "thread", "value": "22052-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_case_01"}], "links": [{"type": "tms", "url": "http://192.168.40.134:8080/projects/CACULATOR/issues/CACULATOR-4?filter=allissues", "name": "用例管理系统"}]}